### ISSUE TITLE
Not working with coinmarketcap: [ 'CoinMarketCap' object has no attribute 'api' ]

### ISSUE THREAD

**Issue by twingall1** [Howchoo/crypto-ticker#24]
![Screenshot 2021-10-01 173619](https://user-images.githubusercontent.com/34135758/135656568-d62267ed-343f-49fb-8bb7-eda73fad9cce.png)

running into this issue when trying to use coinmarketcap API.

My settings.env file:
SYMBOLS=btc
API=coinmarketcap
CMC_API_KEY=XXXXXXXXXXX

Any ideas?

**Comment by snicklefritz36** [comment]
Also experiencing this issue - have played around but not sure how to fix it

**Comment by twingall1** [comment]
yeah tried for a few hours but gave up... since discovered that even if it worked, the limit of 333 on CMC free API would only allow me 1 update every 5 mins. Whereas Coingecko can update every 2mins. Just have to settle for the slightly 'off' BTC price, which never seems to quite agree with the big exchanges.

**Comment by PDM78** [comment]
Would it be possible to use an alternative API, Binance for example?

**Comment by LMDiazDEC20** [comment]
Same here. API not found error.

**Comment by chadgent** [comment]
Same issue here.  @josephtyler Any thoughts on this? Would love to get the CoinMarektCap API working..
